app/components/search-table/resource/row.hbs

```diff
@@ -1,8 +1,8 @@
 <td>
   {{@consumption.subsidyMeasureOffer.title}}
-  {{#if @consumption.subsidyApplicationFlow.subsidyMeasureOfferSeries.title}}
+  {{#if @consumption.subsidyApplicationFlow.subsidyMeasureOfferSeries.label}}
     <AuHelpText @skin="secondary" class="au-u-margin-top-none">
-      ({{@consumption.subsidyApplicationFlow.subsidyMeasureOfferSeries.title}})
```

Comment:
Why did you delete the `(   )`? It's in the mockup @aatauil

Reply:
Ah sorry, found that it looked weird. you can add it back